### PR TITLE
nginx: update to 1.12.1

### PR DESCRIPTION
**Pull request by donvipre**
fixes: CVE-2017-7529

Signed-off-by: Uwe Arnold <donvipre@gmail.com>

### CODE REVIEW

File: net/nginx/Makefile [hunk 1]
```diff
@@ -8,13 +8,13 @@
...
-PKG_VERSION:=1.12.0
+PKG_VERSION:=1.12.1
 PKG_RELEASE:=1
 
 PKG_SOURCE:=nginx-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=http://nginx.org/download/
-PKG_HASH:=b4222e26fdb620a8d3c3a3a8b955e08b713672e1bc5198d1e4f462308a795b30
-PKG_MD5SUM:=995eb0a140455cf0cfc497e5bd7f94b3
+PKG_HASH:=8793bf426485a30f91021b6b945a9fd8a84d87d17b566562c3797aba8fac76fb
+PKG_MD5SUM:=a307e74aca95403e5ee00f153807ce58
```
Comment (pprindeville) [inline net/nginx/Makefile]: This is causing Travis to fail:

`WARNING: Use of PKG_MD5SUM is deprecated, switch to PKG_HASH`

Comment (donvipre) [inline net/nginx/Makefile]: thank you, I'm on it, MD5SUM must be a leftover from previous versions.
I will remove it.

Comment (pprindeville) [inline net/nginx/Makefile]: @heil Do you have eyes on this?

Comment (pprindeville) [inline net/nginx/Makefile]: > thank you, I'm on it, MD5SUM must be a leftover from previous versions.
I will remove it.

It was deprecated earlier this year.

Comment (pprindeville) [inline net/nginx/Makefile]: Oh, also please squash your changes...

**Comment by donvipre** [comment]
any updates on this? how is the procedure to get this merged?

**Comment by pprindeville** [comment]
Still need to hear back from @heil.

**Comment by pprindeville** [comment]
Thanks!

**Pull request merged**